modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-api/src/main/java/com/liferay/portal/search/tuning/rankings/storage/RankingStorageAdapter.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.search.tuning.rankings.storage;
+
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.search.tuning.rankings.index.Ranking;
+import com.liferay.portal.search.tuning.rankings.index.name.RankingIndexName;
+
+/**
+ * @author Almir Ferreira
+ */
```

Comment:
i think most of the interfaces we are creating/migrating should be ProviderType annotated